blocks/separator/index.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    className: 'separator',
+    template: require('./index.html'),
+    data: {
+        name: 'Separator',
```

Comment:
There's some kind of weird whitespace here, but I was trying to keep the same code style as other blocks.